packages/api/src/controllers/experiment/attestation.ts

```diff
@@ -0,0 +1,103 @@
+import { Router } from "express";
+import { v4 as uuid } from "uuid";
+import { db } from "../../store";
+import { validatePost } from "../../middleware";
+import { ethers } from "ethers";
+import { makeNextHREF, toStringValues } from "../helpers";
+import _ from "lodash";
+
+const app = Router();
+
+const DOMAIN = {
+  name: "Verifiable Video",
+  version: "1",
+};
+
+app.post("/", validatePost("attestation"), async (req, res) => {
+  const { primaryType, domain, message, signature } = req.body;
+
+  if (primaryType !== "VideoAttestation") {
+    return res.status(400).json({ errors: ["invalid primaryType"] });
+  }
+  if (!_.isEqual(domain, DOMAIN)) {
+    console.log(domain);
+    console.log(DOMAIN);
+    return res.status(400).json({ errors: ["invalid domain"] });
+  }
+  if (!verifySigner(message, signature)) {
+    return res.status(400).json({ errors: ["invalid signature"] });
+  }
+
+  const attestationMetadata = await db.attestation.create({
+    id: uuid(),
+    createdAt: Date.now(),
```

Comment:
Is there some kind of timestamp in the message as well? If not, that seems like a good addition. If yes, we could consider using that as the `createdAt` value here (especially validating that it has to be recently signed I guess?)